hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/HapiUtils.java

```diff
@@ -89,7 +89,8 @@ private HapiUtils() {}
      */
     public static boolean isHollow(@NonNull final Account account) {
         requireNonNull(account);
-        return (account.accountNumber() > 1000
+        requireNonNull(account.accountId());
```

Comment:
This line is not needed because the very next line will throw if accountID is null.

Reply:
This is a nit that can be fixed in a later PR